### PR TITLE
Feedback: modify feedback layout (fixes #2418)

### LINKED ISSUE #2418: Improve layout of a Feedback view page
## Screenshot
screen shot 2018-10-29 at 10 13 46 pm

### PR DESCRIPTION
**Pull request by svlesiv**
screen shot 2019-02-01 at 6 01 37 pm

screen shot 2019-02-01 at 6 01 50 pm

screen shot 2019-02-01 at 6 02 02 pm

### CODE REVIEW

**Comment by lmmrssa** [comment]
@svlesiv can reply section be made static to always display at bottom of the page and not scroll.
Also we can make message section more responsive.

**Comment by svlesiv** [comment]
Thanks for the review @singharpita and @lmmrssa. Please let me know if more adjustment needed. 

@lmmrssa, what do you mean by 'make message section more responsive'? Have a width of the message card relative to its content? If so, I've fixed that.


screen shot 2019-02-04 at 11 19 55 am


screen shot 2019-02-04 at 11 20 16 am

File: src/app/feedback/feedback-view.scss [hunk 1]
```diff
@@ -38,12 +34,3 @@
...
 }
-
-@media (min-width: $screen-md) {
-  .title-edit {
-    width: 40%;
-  }
-  .view-container {
-    padding: 3vh 25vw;
-  }
-}
```
Comment (svlesiv) [inline src/app/feedback/feedback-view.scss]: Hi @lmmrssa, seems if to delete media, the chat will be too wide on a large screen. You don't think so?

**Comment by svlesiv** [comment]
Hi @lmmrssa, seems if to delete media, the chat will be too wide on a large screen. You don't think so?

**Pull request merged**